AppCenterDistribute/AppCenterDistribute/Resources/zh-Hans.lproj/AppCenterDistribute.strings

```diff
@@ -2,6 +2,7 @@
  * Update dialog buttons
  */
 "MSDistributeUpdateNow" = "立即更新";
+"MSDistributeNotNow" = "现在不要";
```

Comment:
```suggestion
"MSDistributeNotNow" = "不是现在";
```

Reply:
```suggestion
"MSDistributeNotNow" = "现在不更新";
```

My suggestion translates into "Do not update now" which sounds more natural in Chinese language context.